### PR TITLE
[github] show PR controller in PR context

### PR DESCRIPTION
**Pull request by JanKoehnlein**
This PR fixes gitpod-io/gitpod#4081

### CODE REVIEW

**Comment by JanKoehnlein** [comment]
This still lacks the case that the user hasn't granted the necessary scopes in the GitHub access yet. Should we add a notification on this?

**Comment by akosyakov** [comment]
@JanKoehnlein How does it behave in such case? Nothing happens, then it would be good to add a notification.

File: extensions/gitpod/src/extension.ts [hunk 1]
```diff
@@ -1361,13 +1361,29 @@ export async function activate(context: vscode.ExtensionContext) {
...
+
+				// for reasons unknown, the following call to getSession doesn't return if the user still has
+				// to grant access. So we detect the successful login via the change event
+				vscode.authentication.onDidChangeSessions(async e => {
+					if (e.provider.id === 'github') {
+						await vscode.commands.executeCommand('github.api.preloadPullRequest');
+						vscode.commands.executeCommand('workbench.view.extension.github-pull-request');
+					}
+				});
+				await vscode.authentication.getSession('github', ['read:user', 'user:email', 'repo'], { createIfNone: true });
```
Comment (akosyakov) [inline extensions/gitpod/src/extension.ts:1379]: we should a link here where to look up such scopes

**Comment by akosyakov** [comment]
I see this dialog for each worskpace on PR now:
Screenshot 2021-08-04 at 08 20 54

Regardless whether GitHub PR extension already has access rights or not.

**Comment by JanKoehnlein** [comment]
https://github.com/gitpod-io/gitpod/issues/4081#issuecomment-892607493

**Pull request closed without merging**